src/App.css

```diff
@@ -31,7 +31,7 @@ h2 {
 }
```

Comment:
- Haja great job on the assignment. Not only You completed the Todo List but started working on refactor you code and made it better. Amazing job on that!
- Your code is clean and You are properly splitting up the code into multiple components.
- Your git workflow is fluid and You did amazing job on this assignment  👍